packages/flutter/test/rendering/independent_layout_test.dart

```diff
@@ -97,3 +99,10 @@ void main() {
     expect(onscreen.child.size, equals(const Size(800.0, 10.0)));
   });
 }
+
+class FakeFlutterView implements FlutterView {
+  @override
+  dynamic noSuchMethod(Invocation invocation) {
+    return super.noSuchMethod(invocation);
+  }
+}
```

Comment:
If we can use `RendererBinding.instance.platformDispatcher.views.single` where you've used `FakeFlutterView`, we can get rid of the extra implementations of `FlutterView` here and in other files.

Reply:
See other thread.